Sources/MongoSwift/SDAM.swift

```diff
@@ -126,37 +146,29 @@ public struct ServerDescription {
 
         // initialize the rest of the values from the isMaster response.
         // we have to copy because libmongoc owns the pointer.
-        let isMaster = Document(copying: mongoc_server_description_ismaster(description))
-
-        self.lastWriteDate = isMaster["lasWrite"]?.documentValue?["lastWriteDate"]?.dateValue
```

Comment:
>"lasWrite"

whoops

Reply:
compelling argument for using `Codable` 🙂